### DISCUSSION TITLE
Week 6 - Project Task - Craig Butikofer

### DISCUSSION THREAD

**Discussion by CraigButikofer** [USU-ITLS-x265-23-summer/class-repo#216] [General]
I work for a church organization that puts on 5-day camps around the world for teenagers: FSY - For the Strength of Youth. My project ideas are based off projects that I could use for work.

1. Website for In-Area Advisors
2. Website for Training

 Technical Challenges - both projects will have very similar technical challenges
- Formatting- The formatting is really tricky as we have been learning and practicing formatting for the different projects, it is tricky to get all the placements, fonts, backgrounds etc. I will likely reference the freecodecamp projects to get similar formatting and get it to look visually appealing.
- Building- It seems like whomever built the practices out, had a very clear vision of what they wanted the end to look like. I think that I am struggling to know what that vision is in real time as I'm learning what tools I have to use to get there. I'll again likely reference the examples that we have been working on to know exactly how to reach that end goal.
- Troubleshooting- It has been really helpful to be learning in this controlled environment where you get hints when coding goes wrong, but it will be more difficult when I am the person doing the base design. I'll experiment and practice more with my tools so I can troubleshoot along the way to get my end result.

**Comment by derekscholes** [comment]
I have found flexbox to be the easiest to use to format the webpages but I found out too late and only used it a little bit. I suggest playing around with it some to see how you can add it to your site.

**Comment by CJ3315** [comment]
Do you want this site to be more interactive? Or more video based on lectures?
